File: 2022/programmazione/swap/swap.py
#!/bin/env python3

import sys

# Se vuoi leggere/scrivere da file decommenta qua
# fin = open("input.txt", "r")  # File di input fornito dalla piattaforma
# fout = open("output.txt", "w")  # Output da inviare alla piattaforma

# Se vuoi leggere/scrivere da linea di comando decommenta qua
fin = sys.stdin  # input fornito dalla piattaforma
fout = sys.stdout  # Output da inviare alla piattaforma


def swap(N, V):
    # SCRIVI QUA IL TUO CODICE
    return 0


N = int(fin.readline().strip())
V = list(map(int, fin.readline().strip().split(" ")))
print(swap(N, V))

File: 2022/programmazione/swap/swap.java
import java.util.*;
import java.io.File;
import java.io.PrintStream;
import java.io.FileNotFoundException;

public class ppo {

    public static int controlla(int N, int V[]) {
        // SCRIVI QUA IL TUO CODICE
        return 0;
    }

    public static void main(String[] args) throws FileNotFoundException {

        // Se vuoi leggere/scrivere da linea di comando decommenta qua
        Scanner scanner = new Scanner(System.in);

        // Se vuoi leggere/scrivere da file decommenta qua
        // Scanner scanner = new Scanner(new File("input.txt")); // Input fornito dalla piattaforma
        // PrintStream fileStream = new PrintStream("output.txt"); // Output da inviare alla piattaforma
        // System.setOut(fileStream);

        int N = scanner.nextInt();
        scanner.nextLine(); // Move scanner to the next line
        int[] V = new int[N];

        for(int i=0; i<N; i++) {
            V[i] = scanner.nextInt();
        }

        System.out.println(swap(N, V));
    }
}
